### ISSUE TITLE
Function cannot be assigned to Hashable type

### ISSUE THREAD

**Issue by MapleCCC** [microsoft/pylance-release#1027]
## Environment data

-   Language Server version: 2021.2.3
-   OS and version: Windows 18363.1316
-   Python version: vanilla CPython 3.9.0 

## Expected behaviour

In Python, function is hashable. Given that `obj` is a function, the expression `isinstance(obj, collections.abc.Hashable)` evaluates to `True`.

## Actual behaviour

Pylance reports that function cannot be assigned to type "Hashable".

## Code Snippet / Additional information

```python
from collections.abc import Hashable

def calculate_hash(obj: Hashable) -> int:
    return hash(obj)

def increment(x: int) -> int:
    return x + 1

# this statement prints True to the console
print(isinstance(increment, Hashable))

# this statement causes Pylance to report error
calculate_hash(increment)
```

![image](https://user-images.githubusercontent.com/25131775/110611604-54bd4880-81ca-11eb-9818-5cda3fed6038.png)
![image](https://user-images.githubusercontent.com/25131775/110611646-5dae1a00-81ca-11eb-9fae-cd0ced5c432e.png)

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.3.1, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202131-10-march-2021

**Comment by MapleCCC** [comment]
Thanks for your speed work and kind reply. I install the new version and the problem is resolved perfectly. I will close the issue then.
